### PR TITLE
feat: add translations

### PR DESCRIPTION
**Pull request by theguriev**

### CODE REVIEW

**Review by ulianakolodii — APPROVED** [review]

File: src/app/cart/components/Item.tsx [hunk 1]
```diff
@@ -1,39 +1,37 @@
...
+  Partial<
+    Pick<Project, "name" | "blockQuoteEn" | "blockQuoteRu" | "monthlyPrice">
+  > & {
     amount: number;
   }
-> = ({ amount, name, blockQuoteEn, monthlyPrice }) => {
+> = ({ amount, name, blockQuoteEn, blockQuoteRu, monthlyPrice }) => {
   const totalPrice = amount * (monthlyPrice || 0);
+  const locale = getLocale();
+  console.log({
```
Comment (bafxyz) [inline src/app/cart/components/Item.tsx:15]: cleanup

File: src/app/cart/components/Items.tsx [hunk 2]
```diff
@@ -40,36 +40,39 @@ const Items: FC<{
...
 
   return (
     <div className="flex flex-col h-screen bg-tg_secondary_bg_color">
       <div className="flex justify-between py-5 px-6 bg-tg_bg_color">
-        <span className="font-bold text-tg_text_color">YOUR ORDER</span>
+        <span className="font-bold text-tg_text_color">{t("YOUR ORDER")}</span>
         <span className=" text-active">
           <Link href="/" onClick={router.back}>
-            Edit
+            {t("Edit")}
```
Comment (bafxyz) [inline src/app/cart/components/Items.tsx:53]: i would go with such construction, if we will need to connect translation package, not that much to replace

{t`Edit`}

File: src/app/translations.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
+const translations = {
+  en: {
+    Nodes: "Nodes",
+    Testnets: "Testnets",
+    "VIEW ORDER": "VIEW ORDER",
+    PAY: "PAY",
+    "YOUR ORDER": "YOUR ORDER",
```
Comment (bafxyz) [inline src/app/translations.ts:7]: imo better to not use UPPERCASE here, i would use it with tailwing uppercase style

File: src/app/cart/components/Items.tsx (hunk 2 above)
Comment (theguriev) [reply src/app/cart/components/Items.tsx:53]: Then it should be the macro if I understand you correctly.

File: src/app/translations.ts (hunk 3 above)
Comment (theguriev) [reply src/app/translations.ts:7]: I agree. Makes sense! 👍🏻

File: src/app/cart/components/Items.tsx (hunk 2 above)
Comment (bafxyz) [reply src/app/cart/components/Items.tsx:53]: https://cloud.typingmind.com/share/5d60a7e6-a118-4ac5-bc20-9ec1245cc87a i mean like this

File: src/app/components/TelegramScript.tsx [hunk 4]
```diff
@@ -1,17 +1,26 @@
...
-      onLoad={handleOnLoad}
-      defer
-    />
+    <>
+      <Script
+        src="https://telegram.org/js/telegram-web-app.js"
+        onLoad={handleOnLoad}
+        defer
+      />
+      {loaded && children}
```
Comment (bafxyz) [inline src/app/components/TelegramScript.tsx:21]: {loaded ? children : <Skeleton />}

File: src/app/translations.ts (hunk 3 above)
Comment (theguriev) [reply src/app/translations.ts]: Fixed

File: src/app/components/TelegramScript.tsx (hunk 4 above)
Comment (theguriev) [reply src/app/components/TelegramScript.tsx]: Fixed

File: src/app/components/TabsNavigation.tsx [hunk 5]
```diff
@@ -1,35 +1,43 @@
-"use client"
+"use client";
```
Comment (bafxyz) [inline src/app/components/TabsNavigation.tsx:1]: what is use client?

File: src/app/components/TabsNavigation.tsx [hunk 6]
```diff
@@ -1,35 +1,43 @@
...
 import { FC } from "react";
```
Comment (bafxyz) [inline src/app/components/TabsNavigation.tsx:2]:
```js
import { FC } from "react";
import Link from "next/link";
import classnames from "classnames";
import t from "@/app/utils/t";

interface Props {
  path: string;
}

const TabsNavigation: FC<Props> = ({ path }) => {
  const isCurrentTab = (currentPath: string) => {
    return path === currentPath;
  };

  const tabs = [
    {
      path: "/",
      label: t("Nodes"),
    },
    {
      path: "/testnets",
      label: t("Testnets"),
    },
  ];

  return (
    <div className="flex justify-center mb-4 border-b bg-tg_secondary_bg_color">
      <ul
        className="w-[320px] flex flex-wrap -mb-px text-sm font-medium text-center gap-4"
        id="myTab"
        data-tabs-toggle="#myTabContent"
        role="tablist"
      >
        {tabs.map((tab) => (
          <li key={tab.path} className="mr-2" role="presentation">
            <Link href={tab.path} passHref>
              <a
                className={classnames(
                  "inline-block py-4 border-b-2 rounded-t-lg",
                  {
                    "border-solid text-tg_button_color": isCurrentTab(tab.path),
                    "text-tg_hint_color hover:text-gray-600 hover:border-gray-300 dark:hover:text-gray-300":
                      !isCurrentTab(tab.path),
                  }
                )}
              >
                {tab.label}
              </a>
            </Link>
          </li>
        ))}
      </ul>
    </div>
  );
};

export default TabsNavigation;
```

Comment (bafxyz) [reply src/app/components/TabsNavigation.tsx:2]: A bit more readable imo, but it's up to you if you want to modify it

**Review by bafxyz — APPROVED** [review]

File: src/app/cart/components/Items.tsx (hunk 2 above)
Comment (theguriev) [reply src/app/cart/components/Items.tsx:55]: > https://cloud.typingmind.com/share/5d60a7e6-a118-4ac5-bc20-9ec1245cc87a i mean like this

Unfortunately, In this case, you will lose the type hints

File: src/app/components/TabsNavigation.tsx (hunk 5 above)
Comment (theguriev) [reply src/app/components/TabsNavigation.tsx:1]: New fancy next syntax https://nextjs.org/docs/getting-started/react-essentials#the-use-client-directive

File: src/app/cart/components/Items.tsx (hunk 2 above)
Comment (bafxyz) [reply src/app/cart/components/Items.tsx:55]: I see

Comment (bafxyz) [reply src/app/cart/components/Items.tsx:55]: let's leave like this for now

File: src/app/components/TabsNavigation.tsx (hunk 6 above)
Comment (theguriev) [reply src/app/components/TabsNavigation.tsx]: I think it can be even better. Check now! :)

**Pull request merged**